tests/acceptance/features/webUIFiles/fileFolderActionMenu.feature

```diff
@@ -0,0 +1,84 @@
+    When the user picks the row of file "lorem.pdf" in the webUI
+    Then the app-sidebar for file "lorem.pdf" should be visible on the webUI
+    And the following items should be visible in the actions menu on the webUI
+      | items                   |
+      | open in browser         |
+      | download                |
+    And the following accordion items should be visible in the details dialog on the webUI
+      | items                   |
+      | versions                |
+
+  Scenario: select image file to see actions menu options
+    When the user picks the row of file "testavatar.png" in the webUI
+    Then the app-sidebar for file "testavatar.png" should be visible on the webUI
+    And the following items should be visible in the actions menu on the webUI
+      | items                   |
+      | open in mediaviewer     |
+      | download                |
+    And the following accordion items should be visible in the details dialog on the webUI
+      | items                   |
+      | versions                |
+
+  Scenario: observe different actions menu options on selecting different file types or folder
+    Given the app-sidebar for file "lorem.txt" is visible on the webUI
+    When the user picks the row of folder "simple-folder" in the webUI
+    Then the app-sidebar for folder "simple-folder" should be visible on the webUI
+    And the following items should be visible in the actions menu on the webUI
+      | items                     |
+      | open folder               |
+    And the following items should not be visible in the actions menu on the webUI
+      | items                     |
+      | download                  |
+    When the user picks the row of file "lorem.txt" in the webUI
+    Then the app-sidebar for file "lorem.txt" should be visible on the webUI
+    And the following items should be visible in the actions menu on the webUI
+      | items                     |
+      | download                  |
+    And the following items should not be visible in the actions menu on the webUI
+      | items                     |
+      | open folder               |
+    When the user picks the row of file "lorem.pdf" in the webUI
+    Then the app-sidebar for file "lorem.pdf" should be visible on the webUI
+    And the following items should be visible in the actions menu on the webUI
+      | items                     |
+      | open in browser           |
+      | download                  |
+    When the user picks the row of file "testavatar.png" in the webUI
+    Then the app-sidebar for file "testavatar.png" should be visible on the webUI
+    And the following items should be visible in the actions menu on the webUI
+      | items                     |
+      | open in mediaviewer       |
+      | download                  |
+    And the following items should not be visible in the actions menu on the webUI
+      | items                     |
+      | open in browser           |
+    
```

Comment:
Remove this as the tests above and this scenario are similar.

Reply:
Seems similar but what it actually does is when one folder or file is selected then switching to another file types should show appropriate file action options.

We can also keep this scenario only and remove the above ones.